Dockerfile

```diff
@@ -450,6 +453,17 @@ RUN echo "BUILD_BRANCH=${BUILD_BRANCH}"    | tee /build.info \
     && echo "FDD_LENSES_CLI_VERSION=${LC_VERSION}" | tee -a /build.info \
     && sed -e 's/^/FDD_/' /opt/landoop/build.info  | tee -a /build.info
 
+# Add man pages & README.txt
+ADD ./doc/share/lenses /usr/local/share/lenses
+RUN apk add man-pages \
```

Comment:
For some reason you add again man-pages here? And again, it is not needed. :)

Reply:
I had it there to do fast tests between rebuilds, since the apd add entries are quite many steps behind. As it seems it was forgotten there also